Assets/Scripts/Managers/CombatManager.cs

```diff
@@ -289,4 +289,9 @@ public GameState GameState
             }
         }
     }
+
+    public List<PlayerClass> GetPlayers()
+    {
+        return players;
+    }
```

Comment:
Please do not expose the players list, any code that should have knowledge of all the players should be placed inside this class.

Reply:
Ahhh this was for foresight because remember you wanted the manager itself to have access to the hands? Well they need access to the player list. (Also doesn't the AddAttack of EnemyClass need access to the players anyways?)

Reply:
Exposing a raw reference to the list of players like this violates encapsulation and makes it easy to accidentally mangle the state.

Consider whether you really need access to the entire list. If you do, then consider whether you can expose a read-only slice.

Reply:
I agree with the sentiment: it should be a read-only slice.